faunadb/query.py

```diff
@@ -248,10 +248,10 @@ def _params(main_params, optional_params):
   return main_params
 
 
-def _varargs_query(name, values):
+def _varargs(values):
   """
-  Call name with varargs.
-  This ensures that a single value passed is not put in array, so
-  ``query.add(query.var(x))`` will work where ``x`` is a list whose values are to be added.
+  Called on rest arguments.
```

Comment:
'Rest arguments' are known as 'splats' in ruby. There doesn't seem to be a commonly accepted term for these in python.